lib/plugins/aws/provider/awsProvider.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 
+const AWS = require('aws-sdk');
 const BbPromise = require('bluebird');
 const HttpsProxyAgent = require('https-proxy-agent');
 const url = require('url');
-const AWS = require('aws-sdk');
+
+const Naming = require('../lib/naming.js');
```

Comment:
Just curious. Is there a specific reason / convention why this is uppercased (I thought that classes follow this uppercased pattern)?